### PR TITLE
Fixed adding additional_info

### PR DESCRIPTION
**Pull request by salesfelipe**
On splunk the query would look like:

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -183,16 +183,20 @@ export default class SplunkEvents {
...
         screen.availHeight || '-'
       },${location.hostname},` +
       `${location.pathname},${location.protocol.replace(':', '')},${
         location.hash || '-'
-      }"`
-    )
+      }`
+
+    return {
+      additional_info: additionaInfo,
```
Comment (gabrielvincent) [inline src/index.js:198]: Isn't more JavaScriptesque to use came case for object keys? We even use came case for widely-used Splunk fields, such as `workflowType` and `workflowInstance`.

**Review by gabrielvincent — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 2]
```diff
@@ -183,16 +183,20 @@ export default class SplunkEvents {
     }
     let screen = window.screen ? window.screen : {}
     let location = window.location ? window.location : {}
-    return (
-      `additional_info="${navigator.userAgent.replace(/\,/g, ';')},` +
+
+    const additionaInfo =
```
Comment (gabrielvincent) [inline src/index.js:187]:
```suggestion
    const additionalInfo =
```

File: src/index.js (hunk 1 above)
Comment (salesfelipe) [reply src/index.js:198]: Agreed but that would be a breaking change for someone that it was using  it before....

**Review by gabrielvincent — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (gabrielvincent) [reply src/index.js:198]: Ok. So I'd only ask to correct the variable name, which is mistyped `additionaInfo`

**Review by gabrielvincent — APPROVED** [review]

**Pull request merged**